DirectEmployers/MyJobs
# [PD 3149] state category repopulate

## What's this PR do?
PR repopulates the states and categories list inside of new CBO as well as a current CBO's contact

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside the CBO application

## How should this be manually tested?
Pull down repository. When you go to an already created cbo contact if there is a state or category selected for them you should see it populated on the Manage Contact page. If you go to create a new CBO and add the information for a new contact and add categories and state, hit the back button OR go to edit contact details at the end on the review page, you will be able to see what they have saved and selected for the state and categories repopulating.

## Any background context you want to provide?
